src/main/scala/firrtl/Namespace.scala

```diff
@@ -69,7 +67,7 @@ object Namespace {
       case s: Block => s.stmts flatMap buildNamespaceStmt
       case _ => Nil
     }
-    namespace.namespace ++= (m.ports collect { case dec: IsDeclaration => dec.name })
+    namespace.namespace ++= m.ports map (_.name)
```

Comment:
See definition of Port, it mixes in IsDeclaration
https://github.com/ucb-bar/firrtl/blob/master/src/main/scala/firrtl/ir/IR.scala#L407